source/develop/release-management/process/making-an-release.md

```diff
@@ -2,24 +2,168 @@
 title: Making an oVirt release
 category: release-management
 authors:
-  - dneary
-  - jbrooks
+  - sandrobonazzola
 ---
 
+Updated on: 2022-01-20
+
 # Making an oVirt release
 
-For the moment, this is just a link dump - there is lots of information about releases which hasn't been organoised as far as I can tell.
+The purpose of this document is to provide instructions on how to prepare a release for oVirt project followiing the transition to external providers for
+the git hosting and build execution.
+
+## Development phase
+
+During the development of the next oVirt release, patches are being verified using GitHub Actions.
+Once a PR is merged, the pakages are being built by COPR infrastructure within [ovirt/ovirt-master-snapshot](https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/) project.
+
+Manual and automated testing can then be done using the [snapshot composes provided by COPR](/develop/dev-process/install-nightly-snapshot.html).
```

Comment:
Getting 404 when clicking on `snapshot composes provided by COPR`

Reply:
did you check the built site? or just the github preview? works for me

Reply:
Works for me too.